### ISSUE TITLE
GraphQL Endpoint in a multilanguage installation.

### ISSUE THREAD

**Issue by pahy** [RozbehSharahi/graphql3#66]
At first, thank you to bring GraphQL to Typo3.

I'm using the extension in a TYPO3 v12 installation with multiple languages, each accessible via a prefix (e.g., /de/) defined in the siteconfig.yaml. 
I already figured out to make /de/graphql work, but graphiql always tries to access /graphql, which is not available.
Therefore i would need the API endpoints /graphql and /graphiql to be language-independent and accessible without any language prefix.

Could the extension be enhanced to support language-independent API endpoints at /graphql or /graphiql? Alternatively, maybe somebopdy got a hint on how to configure typo3 to achieve this.

Thanks in advance
Patrick

**Comment by RozbehSharahi** [comment]
@pahy 

Thx for the input. Afaik the graphlq endpoint is in fact language-independent and therefore i think it is actually a good idea to not serve on `/{lang}` as you have requested.

i will try to have a look this weekend. Will create a scenario to reproduce the issue and introduce probably some backwards-compatible patch.

kind regards

**Comment by RozbehSharahi** [comment]
Hey @pahy 

I could reproduce your issue. I am pretty sure it was due to your default-language, with a base-url different to `/`.

So i created a setup like this:

```
site: main
baseUrl: /
languages:
  en: 
    baseUrl: '/en/'
  de: 
    baseUrl: '/de/'
```
and by that reproduced the issue you had.

Currently it was much easier simply serve graphql on every language-base-url instead of having one single language-independent endpoint. However, I also adapted graph**I**ql and by that your issue should be solved.

The change is available in version v1.8.1.
